com.unity.render-pipelines.high-definition/Runtime/Lighting/Light/HDLightRenderDatabase.cs

```diff
@@ -0,0 +1,339 @@
+using System;
+using UnityEngine.Jobs;
+using System.Collections.Generic;
+using UnityEngine.Assertions;
+using Unity.Collections;
+using Unity.Mathematics;
+using Unity.Collections.LowLevel.Unsafe;
+
+namespace UnityEngine.Rendering.HighDefinition
+{
+    //Light rendering entity. This struct acts as a handle to set / get light render information into the database.
+    internal struct HDLightRenderEntity
+    {
+        public int entityIndex;
+        public static readonly HDLightRenderEntity Invalid = new HDLightRenderEntity() { entityIndex = HDLightRenderDatabase.InvalidDataIndex };
+        public bool valid { get { return entityIndex != HDLightRenderDatabase.InvalidDataIndex; } }
+    }
+
+    //Data of the lights inside the database.
+    internal struct HDLightRenderData
```

Comment:
As we discussed yesterday, at some point, after this first round, would be worth evaluating the ordering of these fields if not moving to a SoA pattern (though that is likely much more work)

Reply:
Indeed, I have added a comment to express that.